Makefile

```diff
@@ -9,3 +9,6 @@ push-flats:
 	docker build -t belka256/flatsapp:latest .
 	docker push belka256/flatsapp:latest
 
+run:
+	go run cmd/users/main.go
+	go run cmd/web/main.go
```

Comment:
Does `cmd/web/main.go` point to Flats service, if so probably it's better to rename path to `cmd/flats/main.go`